pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -285,6 +294,57 @@ def task_schedules_api_client(pulpcore_client):
         return TaskSchedulesApi(pulpcore_client)
 
 
+@pytest.fixture
+def status_api_client(pulpcore_client):
+    return StatusApi(pulpcore_client)
+
+
+@pytest.fixture
+def stop_and_check_services(status_api_client):
+    """Stop services and wait up to 10 seconds to check if services have stopped."""
+
+    def _stop_and_check_services(pulp_services=None):
```

Comment:
How are the pulp_services determined, if i have a dev-box with 8 workers? (At least with systemd we have a pulpcore service that will (re-)start all pulp processes.)
Maybe we would need such a thing in the container setup first. So this should not hold the PR back.

Reply:
Currently they are hard-coded. I agree there is a room for improvement, outside of this PR.